packages/eslint-plugin/tests/rules/no-misused-promises.test.ts

```diff
@@ -500,780 +500,1804 @@ foo(bar);
       },
       options: [{ checksVoidReturn: { attributes: true } }],
     },
-  ],
+    // #region checksVoidReturn.heritageTypes
```

Comment:
Heh, we don't usually use `#region` tags. IMO the cases should be clear enough from their contents that we shouldn't need them. Regions also tend to get murky and poorly defined over time.

Reply:
Ah, yeah fair enough. I figured `#region`s aren't ideal but could be useful for such a huge list of cases like this, but yeah happy to remove the regions